web/packages/teleport/src/AuthConnectors/ConnectorList/CTAConnectors.tsx

```diff
@@ -0,0 +1,120 @@
+/**
+ * Teleport
+ * Copyright (C) 2024  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import styled from 'styled-components';
+
+import Box from 'design/Box';
+import { ResourceIcon } from 'design/ResourceIcon';
+import { H2, Subtitle2 } from 'design/Text';
+import Flex from 'design/Flex';
+import { RocketLaunch } from 'design/Icon';
+
+import { ButtonLockedFeature } from 'teleport/components/ButtonLockedFeature';
+import { CtaEvent } from 'teleport/services/userEvent';
+
+export default function CTAConnectors() {
```

Comment:
```suggestion
export default function CtaConnectors() {
```

Feel free to fix later since this is a backport.